.gitignore

```diff
@@ -135,3 +135,4 @@ dmypy.json
 
 # Temporary result folder
 results
+temp_results
```

Comment:
Isar-turtlebot specific things should be in isar-turtlebot and not in isar, right?

Reply:
Agreed, but do we have a good way to store the temporary results in isar-turtlebot when the application runs from isar?

Reply:
I agree with @tsundvoll, although there is no easy solution for this really. As such, as its only a `.gitignore` entry I would let it slide.